test/stats.bats

```diff
@@ -24,18 +24,63 @@ function teardown() {
 
     # then
     JSON="$output"
-    echo $JSON | jq -e '.stats[0].attributes.id != ""'
+    run echo $JSON | jq -e '.stats[0].attributes.id != ""'
     [ "$status" -eq 0 ]
 
-    echo $JSON | jq -e '.stats[0].cpu.timestamp > 0'
+    run echo $JSON | jq -e '.stats[0].cpu.timestamp > 0'
     [ "$status" -eq 0 ]
 
-    echo $JSON | jq -e '.stats[0].cpu.usageCoreNanoSeconds.value > 0'
+    run echo $JSON | jq -e '.stats[0].cpu.usageCoreNanoSeconds.value > 0'
     [ "$status" -eq 0 ]
 
-    echo $JSON | jq -e '.stats[0].memory.timestamp > 0'
+    run echo $JSON | jq -e '.stats[0].memory.timestamp > 0'
     [ "$status" -eq 0 ]
 
-    echo $JSON | jq -e '.stats[0].memory.workingSetBytes.value > 0'
+    run echo $JSON | jq -e '.stats[0].memory.workingSetBytes.value > 0'
+    [ "$status" -eq 0 ]
+}
+
+@test "container stats" {
+    # given
+    container2config=$(cat "$TESTDATA"/container_redis.json | python -c 'import json,sys;obj=json.load(sys.stdin);obj["name"] = ["podsandbox1-redis2"];obj["metadata"]["name"] = "podsandbox1-redis2"; json.dump(obj, sys.stdout)')
+    echo "$container2config" > "$TESTDIR"/container_redis2.json
+    run crictl runp "$TESTDATA"/sandbox_config.json
+    echo "$output"
+    [ "$status" -eq 0 ]
+    pod_id="$output"
+    run crictl create "$pod_id" "$TESTDATA"/container_redis.json "$TESTDATA"/sandbox_config.json
+    echo "$output"
+    [ "$status" -eq 0 ]
+    ctr1_id="$output"
+    run crictl create "$pod_id" "$TESTDIR"/container_redis2.json "$TESTDATA"/sandbox_config.json
+    echo "$output"
+    [ "$status" -eq 0 ]
+    ctr2_id="$output"
+    run crictl start "$ctr1_id"
+    echo "$output"
+    [ "$status" -eq 0 ]
+    run crictl start "$ctr2_id"
+    echo "$output"
+    [ "$status" -eq 0 ]
+
+    # when
+    run crictl stats -o json "$ctr1_id"
+    echo "$output"
+    [ "$status" -eq 0 ]
+    ctr1_stats_JSON="$output"
+
+    run crictl stats -o json "$crt2_id"
```

Comment:
bug: `s/crt2_id/ctr2_id/` (same bug as in the original PR)

It works since
1. it is the same as running `crictl stats` without ID, in which case it shows stats for all IDs.
2. the values obtained are not compared in any way, just printed.